apps/web/package.json

```diff
@@ -85,6 +85,7 @@
     "next": "^13.1.1",
     "next-auth": "^4.10.3",
     "next-axiom": "^0.16.0",
+    "next-build-id": "^3.0.0",
```

Comment:
Why is this needed?

Reply:
This package get de last commit hash and uses like build id, when set an build id on nextConfig nextjs uses it in cache path on production